src/main/java/com/synopsys/integration/alert/web/controller/HomeController.java

```diff
@@ -60,17 +64,23 @@ public String index() {
         final HttpServletRequest httpRequest = request;
         final CsrfToken csrfToken = csrfTokenRespository.loadToken(request);
         final Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
-        final boolean hasAlertRole = authorizationManager.hasAlertRole();
-        final boolean authorized = authentication.isAuthenticated() && hasAlertRole && csrfToken != null;
+        final boolean isAnonymous = authentication.getAuthorities().stream()
+                                        .map(authority -> authority.getAuthority())
+                                        .collect(Collectors.toSet())
+                                        .contains("ROLE_ANONYMOUS");
```

Comment:
Where does this role come from? Should it be a constant?

Reply:
It is the default assigned.  I will make it a constant in the class.